DESCRIPTION

```diff
@@ -1,8 +1,11 @@
 Package: rlena
-Title: Functions for Parsing 'ITS' Files from 'LENA'
+Title: rlena makes working with LENA .its files easy
```

Comment:
This will fail the CRAN checks.

> The mandatory ‘Title’ field should give a short description of the package. Some package listings may truncate the title to 65 characters. It should use title case (that is, use capitals for the principal words: tools::toTitleCase can help you with this), not use any markup, not have any continuation lines, and not end in a period (unless part of …). Do not repeat the package name: it is often used prefixed by the name. Refer to other packages and external software in single quotes, and to book titles (and similar) in double quotes.

https://cran.r-project.org/doc/manuals/r-release/R-exts.html